gtfs/spec/en/reference.md

```diff
@@ -32,6 +33,22 @@ This section defines terms that are used throughout this document.
 
+## Field Types
+
+- **Ad-hoc** - Please see the detail column of the field for the definition of the field type.
+- **Color** - The field contains a color encoded as a six-character hexadecimal number. Refer to https://htmlcolorcodes.com/ to generate a valid value. E.g. "FFFFFF" for white, "000000" for black or "0039A6" for the A,C,E lines in NYMTA.
+- **Enum** - The field contains a value among a set of ad-hoc predefined constants defined in the "Details" column. E.g. the route_type field contains a "0" for tram, a "1" for subway...
+- **ID** - The field contains an identifier which uniquely identifies an entity (therefore  is dataset unique). Its value is not aimed to be displayed to the user, and is a sequence of any UTF-8 characters, but using only printable ASCII characters is recommended.
+- **Positive Integer** - The field contains a non-negative whole number (which includes 0).
+- **Language Code** - The field contains a IETF BCP 47 language code. For an introduction to IETF BCP 47, please refer to http://www.rfc-editor.org/rfc/bcp/bcp47.txt and http://www.w3.org/International/articles/language-tags/. E.g. "en" for English, "en-US" for American English or "de" for German.
+- **Latitude** - The field contains the WGS84 latitude as a float number (it includes decimals). The value must be higher and -90.0 and smaller than 90.0.
+- **Longitude** - The field contains the WGS84 longitude as a float number (it includes decimals). The value must be higher and -180.0 and smaller than 180.0.
+- **Service Date** - The field contains the service date in the YYYYMMDD format. Since service time within a service date can be above 24:00:00, a service date often contains information for the subsequent day(s). E.g "20180913" for Septembre 13th, 2018.
+- **Service Time** - The field contains the service time in the HH:MM:SS format (H:MM:SS is also accepted). The time is measured from "noon minus 12h" of the service day (effectively midnight except for days on which daylight savings time changes occur). For times occurring after midnight, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins. E.g. "14:30:00" for 2:30PM or "25:35:00" for 1:35AM on the next date.
+- **Text string** - The field contains a string of UTF-8 caracters, which is aimed to be displayed to the rider and which must therefore be readable.
+- **Timezone** - The field contains a TZ timezone from the TZ database. Timezone names never contain the space character but may contain an underscore. Refer to http://en.wikipedia.org/wiki/List_of_tz_zones for a list of valid values. E.g. "Asia/Tokyo", "America/Los_Angeles" or "Africa/Cairo".
+- **URL** - The field contains a fully qualified URL that includes http:// or https://, and any special characters in the URL must be correctly escaped. See http://www.w3.org/Addressing/URL/4_URI_Recommentations.html for a description of how to create fully qualified URL values.
```

Comment:
To match https://github.com/google/transit/pull/93, I'd use:

`The field contains a fully qualified URL that includes a scheme such as **http**:// or **https**://,...`

Reply:
Wait. I though we were deciding to enlarge the definition only for the real-time feed. Do we want to allow agency_url to not be Http? This would be a change in the Spec.

Reply:
Hmmmm...I see your point.  Ok, let's stick with the original wording for GTFS.  I have a feeling that this particular discussion could derail the proposal so let's keep it simple and stick to the existing definition for now.